.vscode/settings.json

```diff
@@ -6,7 +6,7 @@
     "editor.defaultFormatter": "charliermarsh.ruff",
     "editor.codeActionsOnSave": {
         "source.fixAll": "always",
-        "source.organizeImports": "always"
+        "source.organizeImports": "never"
```

Comment:
Can we remove this change? We want to organize imports for all files by default.

Reply:
Yeah, I can remove it. For some reason when I had it as "always" it kept changing my imports to be incompatible with the formatting tests so it always failed. I will revert this though and just change the config locally on my end.